OasisDEX/oasis-borrow
# Sl ratio validator fix

# [Sl ratio validator fix](https://shortcutLink)

https://www.notion.so/oazo/Bugs-and-Suggestions-3fa8565c715f4c31bcdeb07cc8e9e76b

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- changed validation from `lower than` to `lower than or equal to`
  
## How to test 🧪
  <Please explain how to test your changes>
- setup stop loss to for example 200% and then try to modify vault to push it to coll ratio equal to 200% - in this case error should popup
